Mathlib/MeasureTheory/Function/SimpleFunc.lean

```diff
@@ -1312,6 +1312,56 @@ protected theorem induction {α γ} [MeasurableSpace α] [AddMonoid γ] {P : Sim
     · simp [piecewise_eq_of_not_mem _ _ _ hy, -piecewise_eq_indicator]
 #align measure_theory.simple_func.induction MeasureTheory.SimpleFunc.induction
 
+/-- In a topological vector space, the addition of a measurable function and a simple function is
+measurable. -/
+theorem _root_.Measurable.add_simpleFunc
+    {E : Type*} [MeasurableSpace α] [MeasurableSpace E] [AddGroup E] [MeasurableAdd E]
```

Comment:
Could you turn the MeasurableSpace arguments into implicit arguments? They have to agree with the measurable spaces in `hg`.
Same remark in the next lemma.